pootle/static/js/shared/components/AboutDialog.js

```diff
@@ -31,22 +32,25 @@ const AboutDialogContent = React.createClass({
           <h1>Zing</h1>
 
           <p>
-            This server is powered by <strong>Zing</strong> —
+            {tct(`This server is powered by %(zing)s —
```

Comment:
Note this is not a template string, and regular quotes should be used instead. Also note that the word wrapping would expose extra whitespace after extraction, so it should go in a single line.

The same applies to _KeyboardHelpDialog_.